src/rust-crypto/verification.ts

```diff
@@ -450,15 +454,16 @@ abstract class BaseRustVerifer<InnerType extends RustSdkCryptoJs.Qr | RustSdkCry
 > {
     /** A promise which completes when the verification completes (or rejects when it is cancelled/fails) */
     protected readonly completionPromise: Promise<void>;
+    private onChangeCallback: (() => Promise<void>) | undefined = undefined;
```

Comment:
Given we know that `onChangeCallback` will be set in the constructor, I think there's a way to tell typescript that.

```suggestion
    private onChangeCallback!: () => Promise<void>;
```

or something to that effect, IIRC